operators/aqua/2022.4.1/bundle.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM scratch
```

Comment:
@yossig-aquasec Be aware that the bundle is ignored, and we only use the annotations.yml file.